Essentials/src/com/earth2me/essentials/commands/Commandpotion.java

```diff
@@ -39,28 +39,53 @@ protected void run(final Server server, final User user, final String commandLab
         }
         
         boolean holdingPotion = stack.getType() == Material.POTION;
-        if (!holdingPotion && ReflUtil.getNmsVersionObject().isHigherThanOrEqualTo(ReflUtil.V1_9_R1)) {
+        boolean holdingArrow = false;
```

Comment:
I'd move the logic here to MaterialUtil - perhaps one method for `isTippableArrow` to determine whether they're on 1.9+ or not and also whether the type is a type of arrow that can have a potion applied, and `isSpectralArrow` to determine whether or not it's spectral.